tests/foreman/ui_airgun/test_settings.py

```diff
@@ -193,154 +195,19 @@ def test_positive_update_authorize_login_delegation_param(session, set_original_
             assert result['table'][0]['Value'] == param_value
 
 
-@tier1
-@upgrade
-def test_positive_update_administrator_param(session, set_original_property_value):
-    """Updates parameter "administrator" under General tab
-
-    :id: c3d53354-b190-4beb-94b7-d2c6e5759608
-
-    :expectedresults: Parameter is updated successfully
-
-    :CaseImportance: Critical
-    """
-    property_name = 'administrator'
-    set_original_property_value(property_name)
-    with session:
-        for param_value in valid_settings_values():
-            session.settings.update(
-                'name = {}'.format(property_name),
-                param_value
-            )
-            result = session.settings.read('name = {}'.format(property_name))
-            assert result['table'][0]['Value'] == param_value
-
-
-@tier1
-def test_positive_update_authorize_login_delegation_api_param(session,
-                                                              set_original_property_value):
-    """Updates parameter "authorize_login_delegation_api" under Auth tab
-
-    :id: 70245d20-c940-40c6-bf3a-80127fb81758
-
-    :expectedresults: Parameter is updated successfully
-
-    :CaseImportance: Critical
-    """
-    property_name = 'authorize_login_delegation_api'
-    set_original_property_value(property_name)
-    with session:
-        for param_value in valid_boolean_values():
-            session.settings.update(
-                'name = {}'.format(property_name),
-                param_value
-            )
-            result = session.settings.read('name = {}'.format(property_name))
-            assert result['table'][0]['Value'] == param_value
-
-
-@tier1
-def test_negative_update_entries_per_page_param(session, set_original_property_value):
+@tier2
```

Comment:
It more looks a tier1 test

Reply:
I would leave only one negative scenario only. We should not cover negative scenario at all, but let's keep one

Reply:
@oshtaier @vijay8451 As this a negative test and tier1 test. As per your earlier comments, I will be removing this test as well. Please let me know if you have questions

Reply:
@omkarkhatavkar I would say to remove all tier1 tests here ... but as mentioned by @oshtaier he is good to keep one negative test .

@oshtaier what do you say on it ? as we could cover negative tests under `API/CLI` around tier1 ?

Reply:
@vijay8451 Just to add here this, such tests (invalid input/validation tests) should be handled in a unit test level. If you consider this test this test validate the UI Error for invalid input (So intention to see javascript error after an ajax call exception ). Certainly, this is cannot be validated in API/CLI.

Reply:
@omkarkhatavkar right , then as per my understanding we should keep one nagative test and keep it with  `tier2`  marker (as exception one by mentioning in doc string) ... what do you think ?
@oshtaier ^^

Reply:
@vijay8451 @oshtaier we do have tier2 tests which are actually tier1 test similar to here https://github.com/omkarkhatavkar/robottelo/blob/9e5c7f97831c58e3ec8a27639a13c89b54bc37ad/tests/foreman/ui_airgun/test_contentview.py#L865

Reply:
that is right, we have few rare situation, so yeah, keep tier2 marker for that one case

Reply:
@omkarkhatavkar then I believe not an issue to add one negative test :+1: